<section id="{{ page.section-type }}" class="container content-section text-center">
<h3>List of Projects</h3>
<div class="row">
    <div class="col-md-10 col-md-offset-1" align="left">
<ul>
<li>
  <a href="https://github.com/splatlab/mantis/tree/mergeMSTs">Mantis</a>
  <small>
: is a space and time efficient data structure to index and query large collections of raw sequencing read experiments. The index is based on colored de Bruijn graph representation and therefore supports graph-based operations such as graph traversal, and bubble calling useful for assembly and variation detection. In our recent work we have advanced the index to more than 30,000 raw read sequencing samples and enabled the nice feature of gradual growth by making the index incrementally updatable.
  </small>
</li>
<li>
  <a href="https://github.com/COMBINE-lab/pufferfish/tree/develop">Pufferfish+Puffaligner</a>
  <small>
: This tool provides an efficient data structure for indexing colored compacted de Bruijn graphs which can achieve a balance between time and space resources by making use of succinct data structures and minimum perfect hash function. Pufferfish provides the underlying data structure for mapping short sequencing reads to a huge population of references while keeping the mapping information for each reference individually. Submitted to Recomb 2018. 
PuffAligner, our recent work, is a highly sensitive aligner on top of Pufferfish for aligning different types of short sequencing reads to a huge population of references, specifically good in the representation of high similarity in the reference sequences.
  </small>
</li>
<li>
  <a href="https://github.com/COMBINE-lab/rainbowfish">Rainbowfish</a>
  <small>
: This tool provides a new data structure to store and query colored de Bruijn graphs that in case of large data sets improves storage by more than twenty times compared to state-of-the-art tools without hurting performance of the queries.
  </small>
</li>
<li>
  <a href="https://github.com/COMBINE-lab/grouper">Grouper</a>
  <small>
: which is an extension on RapClust, is a tool for clustering contigs of a de novo transcriptome assembly. We improved the accuracy of clustering by making use of orphan reads, for which each end of the pair is mapped to a different reference sequence.
  </small>
</li>
<li>
  <a href="https://github.com/fataltes/feature_distribution">MLDD</a>
  <small>
  : Multi-Level Distribution Detection project (MLDD) is the one I worked on when I was in Data Science and NLP lab in 2015. Using statistical tests and classification models such as NaiveBayes we show how distribution of NLP features in social media changes in different levels of analysis (county, user, and message). This can highly affect prior assumptions for further text analysis as we show that central-limit theorem could be applied in social media language analysis as well.
</small>
</li>
<li>
  <font color="MediumSeaGreen">AutismFD</font>
  <small>
  : In this project I designed a game application to improve face emotion detection in children with Autism. Beside collaboration with psychology students to design the method, I also implemented the idea as a tool in C# language. This package was used in a treatment center to help children with Autism to identify face emotions and track their progress over time. 
</small>
</li>
<li>
  <a href="https://github.com/COMBINE-lab/salmon/tree/factorizations">DDFactorization</a>
  <small>
 : We design a new efficient and accurate factorization of likelihoods across equivalence classes of read fragments in <a href ="https://github.com/COMBINE-lab/salmon">Salmon</a>, a transcript-level quantification estimation tool. Using the new model, the estimation accuracy improves significantly without hurting the run time.
</small>
</li>
</ul>
</div>
</div>
</section>
